kafkakewl-metrics/src/main/scala/com/mwam/kafkakewl/metrics/services/HighOffsetMetricsExposer.scala

```diff
@@ -93,29 +130,40 @@ case class MetricMap(metricMap: Map[String, (Map[Int, (Metric.Counter[Long], Lon
   }
 }
 
+class HighOffsetMetricsExposer
```

Comment:
Maybe add a comment here, otherwise this looks weird (I know I suggested to make it a proper ZLayer / service, but you are right, at the moment there is nothing in it really).